bigchaindb/common/transaction.py

```diff
@@ -1272,17 +1271,34 @@ def from_dict(cls, tx_body):
 
         if proposed_tx_id != valid_tx_id:
             raise InvalidHash()
+
+        try:
```

Comment:
Might be worth putting this in the common.schema model, maybe as `validate_transaction_schema`?